units/src/fee_rate/mod.rs

```diff
@@ -408,9 +408,9 @@ mod tests {
         assert_eq!(fee_rate.to_sat_per_kvb_ceil(), 2_001);
 
         let max = FeeRate::MAX;
-        assert_eq!(max.to_sat_per_kwu_ceil(), u64::MAX / 4_000);
-        assert_eq!(max.to_sat_per_vb_ceil(), u64::MAX / 1_000_000);
-        assert_eq!(max.to_sat_per_kvb_ceil(), u64::MAX / 1_000);
+        assert_eq!(max.to_sat_per_kwu_ceil(), u64::MAX / 4_000 + 1);
+        assert_eq!(max.to_sat_per_vb_ceil(), u64::MAX / 1_000_000 + 1);
+        assert_eq!(max.to_sat_per_kvb_ceil(), u64::MAX / 1_000 + 1);
```

Comment:
So our ceil functions weren't actually ceil?

In 3bdc48f5e5ebd09b76fb05b7cce101fe3053a5be - props for remembering these.

Reply:
Correct.